fog/view/server/src/router_admin_service.rs

```diff
@@ -62,7 +66,11 @@ impl FogViewRouterAdminService {
             ChannelBuilder::default_channel_builder(grpc_env)
                 .connect_to_uri(&view_store_uri, logger),
         );
-        shard_clients.insert(view_store_uri, Arc::new(view_store_client));
+        // TODO: Add block range or sharding strategy to this...
```

Comment:
I think this PR would resolve this TODO?

Reply:
Done.